src/main/java/com/amuzil/omegasource/magus/skill/modifiers/ModifierData.java

```diff
@@ -0,0 +1,19 @@
+package com.amuzil.omegasource.magus.skill.modifiers;
+
+import net.minecraftforge.common.util.INBTSerializable;
+
+public abstract class ModifierData implements INBTSerializable {
```

Comment:
Look into making this a `DataTrait` or something similar. While `DataTraits` are yet to be rewritten by Maht, it is how we'll be storing data for entities.